src/inet.c

```diff
@@ -670,7 +670,7 @@ int listen_unix(char *file) {
 		return -1;
 	}
 	saun.sun_family = AF_UNIX;
-	strcpy(saun.sun_path, file);
+	strncpy(saun.sun_path, file, sizeof(saun.sun_path) - 1);
```

Comment:
why is it -1 ?

Reply:
If  there  is  no  terminating null byte in the first n bytes of src, strncpy() produces an unterminated string in
       dest.  If buf has length buflen, you can force termination using something like the following:
```
           strncpy(buf, str, buflen - 1);
           if (buflen > 0)
               buf[buflen - 1]= '\0';
```